--- gtk-sharp-2.5.91/configure.in.orig	2005-10-14 13:09:21.000000000 +0200
+++ gtk-sharp-2.5.91/configure.in	2005-10-14 13:13:32.000000000 +0200
@@ -87,6 +87,7 @@
 
 MONO_REQUIRED_VERSION=1.0
 PKG_CHECK_MODULES(MONO_DEPENDENCY, mono >= $MONO_REQUIRED_VERSION, has_mono=true, has_mono=false)
+PKG_CHECK_MODULES(MINT_DEPENDENCY, mint >= $MONO_REQUIRED_VERSION, has_mint=true, has_mint=false)
 
 AC_PATH_PROG(GACUTIL, gacutil, no)
 if test "x$GACUTIL" = "xno" ; then
@@ -111,6 +112,20 @@
 else
 	enable_dotnet=no
 fi
+elif test "x$has_mint" = "xtrue"; then
+dnl use mint if mono is not found
+GACUTIL_FLAGS='/package $(PACKAGE_VERSION) /root $(DESTDIR)$(prefix)/lib'
+GENERATED_SOURCES=generated/*.cs
+AC_PATH_PROG(RUNTIME, mint, no)
+AC_PATH_PROG(CSC, mcs, no)
+LIB_PREFIX=.so
+LIB_SUFFIX=
+SDCHECK="`$GACUTIL /l |grep ^System.Drawing, | head -n1 |cut -f1 -d','`"
+if test "x$SDCHECK" = "xSystem.Drawing"; then
+	enable_dotnet=yes
+else
+	enable_dotnet=no
+fi
 else
 AC_PATH_PROG(CSC, csc.exe, no)
 GACUTIL_FLAGS=
