--- gtk-sharp-2.12.45/configure.in.orig	2026-05-21 01:18:42.521388608 +0200
+++ gtk-sharp-2.12.45/configure.in	2026-05-21 01:18:42.633351580 +0200
@@ -108,6 +108,7 @@
 MONO_REQUIRED_VERSION=1.0
 FIRST_MONO_VERSION_WITH_NET_4_6_SUPPORT=4.4
 PKG_CHECK_MODULES(MONO_DEPENDENCY, mono >= $MONO_REQUIRED_VERSION, has_mono=true, has_mono=false)
+PKG_CHECK_MODULES(MINT_DEPENDENCY, mint >= $MONO_REQUIRED_VERSION, has_mint=true, has_mint=false)
 if test "x$has_mono" = "xfalse" ; then
 	PKG_CHECK_MODULES(MONO_DEPENDENCY, mono-2 >= $MONO_REQUIRED_VERSION, has_mono=true, has_mono=false)
 	if test "x$has_mono" = "xtrue" ; then
@@ -155,6 +156,20 @@
 else
 	enable_dotnet=no
 fi
+elif test "x$has_mint" = "xtrue"; then
+dnl use mint if mono is not found
+GACUTIL_FLAGS='/package $(PACKAGE_VERSION) /root $(DESTDIR)$(prefix)/lib'
+GENERATED_SOURCES=generated/*.cs
+AC_PATH_PROG(RUNTIME, mint, no)
+AC_PATH_PROG(CSC, mcs, no)
+LIB_PREFIX=.so
+LIB_SUFFIX=
+SDCHECK="`$GACUTIL /l |grep ^System.Drawing, | head -n1 |cut -f1 -d','`"
+if test "x$SDCHECK" = "xSystem.Drawing"; then
+       enable_dotnet=yes
+else
+       enable_dotnet=no
+fi
 else
 AC_PATH_PROG(CSC, csc.exe, no)
 GACUTIL_FLAGS=
