--- gtk-sharp-2.12.10/configure.in.orig	2010-08-15 10:39:28.662820781 +0200
+++ gtk-sharp-2.12.10/configure.in	2010-08-15 10:45:43.986819943 +0200
@@ -80,12 +80,12 @@
 if test "x$platform_win32" = "xyes"; then
 	AC_PATH_PROG(WIX, candle, no)
 	if test "x$WIX" = "xno" ; then
-		enable_msi = no
+		enable_msi=no
 	else
-		enable_msi = yes
+		enable_msi=yes
 	fi
 else
-	enable_msi = no
+	enable_msi=no
 fi
 
 CSDEFINES='-define:GTK_SHARP_2_6 -define:GTK_SHARP_2_8 -define:GTK_SHARP_2_10 -define:GTK_SHARP_2_12'
